main_test.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"os/exec"
+	"testing"
+)
+
+func TestStartServer(t *testing.T) {
```

Comment:
added some tests.  I did the testing using the vanilla go tools.  I didn't think it was worth bringing in additional tools as the tests are pretty simple, but happy to use a tool like testify (if you think it is worth it).

Reply:
Yeah, I would probably use testify, so that on line 31 you can use `t.NoErrorf`

Reply:
for now, let's avoid the extra dependency. Seems like a big one for calling one function.